sklearn/datasets/_svmlight_format_io.py

```diff
@@ -512,6 +512,12 @@ def dump_svmlight_file(
             raise ValueError(
                 "expected query_id of shape (n_samples,), got %r" % (query_id.shape,)
             )
+    else:
+        # NOTE: query_id is passed to Cython functions using fuse type.
```

Comment:
```suggestion
        # NOTE: query_id is passed to Cython functions using a fused type.
```

Reply:
or did you mean a typed memory view?

Reply:
No, you are right: that's a typo from me. :+1: